chat-android/src/main/java/com/ably/chat/RoomOptions.kt

```diff
@@ -127,10 +201,10 @@ internal fun RoomOptions.messagesChannelOptions(): ChannelOptions {
                 add(ChannelMode.publish)
                 add(ChannelMode.subscribe)
 
-                if (presence.enter) {
+                if (it.enter) {
```

Comment:
same goes here

Reply:
When reading it's easy to grasp that `enter` property belongs to `presence`

Reply:
replaced with presence